sw/host/opentitantool/src/command/console.rs

```diff
@@ -97,8 +97,8 @@ impl CommandDispatch for Console {
                 uart.write(send.as_bytes())?;
             }
             if !self.non_interactive {
-                eprintln!("Starting interactive console");
-                eprintln!("[CTRL+C] to exit.\n");
+                eprint!("Starting interactive console\r\n");
+                eprint!("[CTRL+C] to exit.\r\n\r\n");
```

Comment:
Why do we need to use CRLF here?

Reply:
This is from #21052.  We need to use CRLF because at this point, the linux console/terminal is in raw mode rather than cooked mode.  In raw mode, the tty won't translate newline into a carriage-return newline sequence.

Reply:
Ah, thanks